.github/workflows/build_assets.yml

```diff
@@ -61,6 +61,7 @@ jobs:
         run: |
           ${{matrix.CMD_REQS}}
           python setup.py build
+          pip install .
```

Comment:
What does this do?...

Reply:
It installs the CLI as the code in the repo (versus on pip or something else)

Reply:
if `codecov-cli` is not installed the package metadata does not exist so we can't copy it over. By installing it, pyinstaller is able to find the metadata to copy over

Reply:
I see... thank you both for the explanations